### ISSUE TITLE
Not all Pseudo parameters are supported

### ISSUE THREAD

**Issue by mrgrain** [cdklabs/decdk#213]
These are currently not parsed differently.

--- 

https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/pseudo-parameter-reference.html

```
export const PSEUDO_PARAMETER_NAMES = [
  'AWS::AccountId',
  'AWS::Region',
  'AWS::Partition',
  'AWS::URLSuffix',
  'AWS::NotificationARNs',
  'AWS::StackId',
  'AWS::StackName',
  'AWS::NoValue',
];
```

**Comment by mrgrain** [comment]
@otaviomacedo and I have been flip-flopping on whether this should live in the parser or evaluator. Either way, it's currently not supported. Blocked by this PR #215 and needs a revisit then. Adding explicit test cases for all pseudo paramters.

**Comment by mrgrain** [comment]
Implemented as part of #230
